marlowe-runtime/marlowe-tx/Main.hs

```diff
@@ -244,6 +247,14 @@ getOptions = execParser $ info (helper <*> parser) infoMod
       , help "The logging configuration JSON file."
       ]
 
+    submitConfirmationBlocksParser = option (BlockNo <$> auto) $ mconcat
+      [ long "submit-confirmation-blocks"
+      , value 0
```

Comment:
Yes, the default should be zero because some clients will want to tune this themselves.

Reply:
It also preserves the current behaviour by default